async-query-core/src/main/java/org/opensearch/sql/spark/metrics/EmrMetrics.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.metrics;
+
+public enum EmrMetrics {
+  EMR_CANCEL_JOB_REQUEST_FAILURE_COUNT,
+  EMR_GET_JOB_RESULT_FAILURE_COUNT,
+  EMR_START_JOB_REQUEST_FAILURE_COUNT,
+  EMR_INTERACTIVE_QUERY_JOBS_CREATION_COUNT,
+  EMR_STREAMING_QUERY_JOBS_CREATION_COUNT,
+  EMR_BATCH_QUERY_JOBS_CREATION_COUNT;
+}
```

Comment:
The last three metrics relate specifically to the dispatcher, not directly to EMR APIs. It may be beneficial to group these separately from the EMR API failure metrics and consider removing the 'EMR_' prefix to avoid confusion and maintain clarity in naming conventions.

Reply:
I think those metrics are already used ones. Do you know if it is OK to change the metrics name?

Reply:
Lets only do refactoring..changing names we can handle in a different PR.